README.md

```diff
@@ -50,10 +50,6 @@ Currently, Avalanche mainnet and testnet (fuji) are supported. This means that d
 
 To use this project with other chains, the block explorer API key for your target chain must be set in .env (e.g. `SNOWTRACE_KEY` for Avalanche).
 
-An example deployment command looks like:
-
-`yarn deploy --network fuji`
```

Comment:
Is this no longer true?

Reply:
yeah, `yarn deploy` isn't a command anymore

Reply:
`yarn hardhat deploy`

Reply:
so really this line should read:

```
yarn hardhat deploy --network fuji --deployment usdc
```

Reply:
updated